paymentsheet/src/main/java/com/stripe/android/paymentsheet/specifications/BancontactSpec.kt

```diff
@@ -15,7 +15,7 @@ internal val bancontactEmailSection =
     FormItemSpec.SectionSpec(IdentifierSpec("email"), SectionFieldSpec.Email)
 internal val bancontactMandate = FormItemSpec.MandateTextSpec(
     IdentifierSpec("mandate"),
-    R.string.sofort_mandate,
+    R.string.stripe_paymentsheet_sepa_mandate,
```

Comment:
I think this is or should be reused in core.

Reply:
We don't show the SEPA mandate anywhere in `payments-core`. We do have a BECS mandate which is different.